include/flexflow/ops/softmax.h

```diff
@@ -63,28 +43,9 @@ class Softmax : public Op {
                             std::vector<Legion::PhysicalRegion> const &regions,
                             Legion::Context ctx,
                             Legion::Runtime *runtime);
-  void init_meta(SoftmaxMeta *m,
```

Comment:
What happened to `init_meta`? I see the declaration was removed but am not seeing any other changes related to it?

Reply:
I believe the declaration to be obsolete, as I could not find the corresponding implementation of this function.